.github/workflows/CI.yml

```diff
@@ -31,6 +31,8 @@ jobs:
 
       - name: Compile the conventions solution
         run: dotnet build --configuration Release conventions.sln
+      - name: Sanity Check
+        run: dotnet fsi scripts/sanitycheck.fsx
```

Comment:
@Mersho in geewallet, the step is simply called "sanity check" because it does many things; but if we merge this PR here in the conventions repo as it is now, it is only doing one thing: sanity check nuget packages; therefore let's rename the script to sanityCheckNuget.fsx (and rename the CI step too)

Reply:
@Mersho also, this is not a good job to place this step; this job is called "Build" but this is not related to building